Submissions

jeremiasbaur opened the pull request.

jeremiasbaur committed:
OT4

jeremiasbaur committed:
OT3

jeremiasbaur committed:
OT2

jeremiasbaur committed:
OT1

jeremiasbaur committed:
Tutorial

jeremiasbaur committed:
B4 challenge

jeremiasbaur committed:
B1 challenge

pablozsc commented:
hello jeremiasbaur. This is moderator Pablo from Concordium.

We noticed that you filled out the form 2 times and provided us with 2 different wallets. However, you didn't leave a "From submitted" comment in your pull request, as requested on the form. Do it please.
The main purpose of this message is to try to reveal your valid and correct wallet address.
We got 2 entries:

1. 3y7...ny6
2. 3oA...75x

Please tell me which address is correct. This is required for testnet4 rewards to be credited.

jeremiasbaur commented:
Form Submitted Central European Summer Time Time zone in Zürich (GMT+2) Monday, 16 May 2022, 12:14

jeremiasbaur commented:
Hello Pablo

I misunderstood the instructions and added the timestamp to my Github
profile in the profile description: https://github.com/jeremiasbaur
<https://github.com/jeremiasbaur>
I added the timestamp as a comment to the pull request.
My wallet address is the first
one: 3y7mVCc8Gsiavc3ntyP7GSADzzEGc9oKebY8que5pzcKEwvny6

Best regards,
Jeremias


Jeremias Baur
+41 79 516 28 01 <+41795162801>

***@***.***
jeremiasbaur.com <https://jeremiasbaur.com/?utm_source=email_footer>


Am Mo., 23. Mai 2022 um 18:39 Uhr schrieb pablozsc ***@***.***
>:

> hello jeremiasbaur. This is moderator Pablo from Concordium.
>
> We noticed that you filled out the form 2 times and provided us with 2
> different wallets. However, you didn't leave a "From submitted" comment in
> your pull request, as requested on the form. Do it please.
> The main purpose of this message is to try to reveal your valid and
> correct wallet address.
> We got 2 entries:
>
>    1. 3y7...ny6
>    2. 3oA...75x
>
> Please tell me which address is correct. This is required for testnet4
> rewards to be credited.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Concordium/Testnet4-Challenges/pull/481#issuecomment-1134899131>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ4ZFKDNUBZDFF2JBMZFLETVLOYC5ANCNFSM4X7LS63Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
